assets/dev/js/editor/elements/views/container.js

```diff
@@ -407,6 +413,7 @@ const ContainerView = BaseElementView.extend( {
 		setTimeout( () => {
 			this.nestingLevel = this.getNestingLevel();
 			this.$el[ 0 ].dataset.nestingLevel = this.nestingLevel;
+			this.model.set( 'isInner', this.checkIfParentIsWidget() || this.getNestingLevel() > 0 );
```

Comment:
@ArmanPurtoian Can you please double-check that this doesn't break any of the DnD functionality inside the Nested Widgets?